packages/drivekit_core/flutter_drivekit_core_platform_interface/lib/flutter_drivekit_core_platform_interface.dart

```diff
@@ -37,13 +37,16 @@ abstract class DrivekitCorePlatform extends PlatformInterface {
   /// Identifies the user with the given id.
   Future<void> setUserId(String userId);
 
+  /// Get the userId.
+  Future<String?> getUserId();
+
   /// Resets the DriveKit SDK.
   Future<void> reset();
 
   /// Check the validity of the generated token
   /// after connecting with API Key and User ID
   Future<bool> isTokenValid();
 
-  /// Deletes a driver's account
+  /// Deletes a driver's account.
```

Comment:
Just a small remark, if you could put this type of change in a separate commit that would make the review easier 😁

Reply:
Ok, sorry! I'll do it the next time :)